vent/menus/main.py

```diff
@@ -221,7 +221,20 @@ def popup(original_containers, thr, title):
     def system_commands(self, action):
         """ Perform system commands """
         if action == "reset":
-            # !! TODO
+            okay = npyscreen.notify_ok_cancel("This will remove ALL of Vent's containers and images. Are you sure?",
+                                        title="Confirm system command")
+            if okay:
+                # remove containers
+                d_cli = docker.from_env()
+                list = d_cli.containers.list(filters={'label':'vent'})
```

Comment:
I think you're also going to need `all=True` for containers that are no longer running.